flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/agg/GroupingSetsTest.scala

```diff
@@ -354,6 +355,8 @@ class GroupingSetsTest extends TableTestBase {
 
   @Test
   def testCALCITE1824(): Unit = {
+    expectedException.expect(classOf[TableException])
```

Comment:
is grouping sets with same group (`GROUPING SETS ((), ())`) not supported ?

Reply:
Because of CALCITE-1824 has been fixed, the plan with `GROUP_ID` has changed, our blink-planner can not translate that yet. Before CALCITE-1824, `GROUP_ID` returns wrong result.